### ISSUE TITLE
Release 3.2.1

### ISSUE THREAD

**Issue by nilsnolde** [valhalla/valhalla#3870]
IMO we can prepare a new release. There's been a few nice improvements and bug fixes worth releasing and not much in terms of PRs which are worth waiting for I believe, except probably #3585. 

Only a patch update, since no major features were implemented since 3.2.0. Once GTFS or Chinese Postman are merged, it could be 3.3.0?

What do you think?

**Comment by kevinkreiser** [comment]
i dont feel strongly that merging the GTFS or Chinese Postman prs would necessitate a minor version update. mainly because both will likely be in a partially working/beta state as opposed to a proper v1 with no known bugs or major lacking issues. i dont really feel strongly about this though, just a point for discussion that minor version updates could be used to signal completion of significant features rather than beta releases. who knows maybe once we are done with them the work will be in the fully completed state (though i doubt it haha).

**Comment by nilsnolde** [comment]
> just a point for discussion that minor version updates could be used to signal completion of significant features rather than beta releases

True. But then I wonder if we ever think "Ah, now it's good enough, let's increase the minor version for that now stable feature", which I doubt :smile:  I hardly ever do things like `rc` or alpha/beta releases, but I guess that could be a way how to deal with that? Not sure if we'd like to "complicate" things to that point though.

**Comment by nilsnolde** [comment]
I don't feel strongly about it either. I understand what you mean, fine with me.

> Forward-compatible C++ library API

Yeah, seems that needs an update :smile:

**Comment by kevinkreiser** [comment]
> Yeah, seems that needs an update

hahah yeah my thoughts exactly!

**Comment by ImreSamu** [comment]
And please update the localization files !  
Transifex -> https://github.com/valhalla/valhalla/tree/master/locales

**Comment by kevinkreiser** [comment]
@danpaz was mainly handling that but has since moved on to other projects. i'll get the transiflex info from him and see what kinds of updates we have over there. apologies for the delay on that one @ImreSamu

**Comment by kevinkreiser** [comment]
@ImreSamu it looks like i have access already ive just completley forgotten about it. give me a little bit to start pr'ing these updates! sorry again for the wait!

**Comment by nilsnolde** [comment]
@ImreSamu I'm just going through transifex and updated the British & German translations. There's a few more regarding indoor routing. We'll cut a release very soon, so if you want Bulgarian to be up-to-date, I'd recommend updating it soon:)

**Comment by kevinkreiser** [comment]
this is  done
